nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/property-table/property-table.component.ts

```diff
@@ -396,7 +398,8 @@ export class PropertyTable implements AfterViewInit, ControlValueAccessor {
 
     getPropertyTipData(item: PropertyItem): PropertyTipInput {
         return {
-            descriptor: item.descriptor
+            descriptor: item.descriptor,
+            propertyHistory: this.propertyHistory?.propertyHistory[item.property]
```

Comment:
Not really an issue with this code exactly, but the closest place to make the comment contextual...

We only display the tooltip if the property `hasInfo`. In my case, I have a CS with a dynamic property and it fails that test so the history is inaccessible via tooltip. I think we might want to add a new boolean helper like:
```
    hasHistory(item: PropertyItem): boolean {
        return !!this.propertyHistory?.propertyHistory[item.property];
    }
```

then update the logic in the template to be something like:
```
                            <div>
                                @if (hasInfo(item.descriptor) || hasHistory(item)) {
                                    <div
                                        class="fa fa-question-circle primary-color"
                                        nifiTooltip
                                        [tooltipComponentType]="PropertyTip"
                                        [tooltipInputData]="getPropertyTipData(item)"
                                        [delayClose]="false"></div>
                                }
                            </div>

```

It looks like this will solve the same issue for all of the various Controller Settings configuration dialogs